server.js

```diff
@@ -1,32 +1,57 @@
 var Discord = require("discord.js");
 var bot = new Discord.Client();
-var request = require("request");
+var mtrAggregator = new require("./aggregators/rules/mtr.js");
+var ipgAggregator = new require("./aggregators/rules/ipg.js");
+var crAggregator = new require("./aggregators/rules/cr.js");
+var jarAggregator = new require("./aggregators/rules/jar.js");
+var cardAggregator = new require("./aggregators/mtg_cards/cards.js");
 
-var cardApi = "https://api.magicthegathering.io/v1/cards?name=";
+const mtr = "!mtr";
+const ipg = "!ipg";
+const cr = "!cr";
+const jar = "!jar";
+const card = "!card";
+const help = "!help";
 
+/**
+ * @param msg.channel.sendMessage
+ */
 bot.on("message", msg => {
-	if(bot.user.id == msg.author.id) return;
-	var query = "";
-    if (query = msg.content.match(/^!([a-z0-9].+)/i)) {
-		query = encodeURIComponent(query[1]);
-		request({
-  		  url: cardApi + query,
-		  json: true
-		}, (error, response, body) => {
-			if (!error && response.statusCode === 200 && body.cards && body.cards.length) {
-				var card = body.cards[0];
-				var response = ["**"+card.name+"**"];
-				response.push(card.type);
-				response.push(card.text.replace(/\*/g,'\\*'));
-				if(card.loyalty) response.push(card.loyalty);
-				if(card.power) response.push(card.power.replace(/\*/g,'\\*') + "/" + card.toughness.replace(/\*/g,'\\*'));
-				response.push(card.printings.join(", "));
-				response.push("http://magiccards.info/query?q=!"+encodeURIComponent(card.name));
-				if(card.imageUrl) response.push(card.imageUrl);
-				msg.channel.sendMessage(response.join("\n"));
-		    }
-		});
-    }
+	if(bot.user.id === msg.author.id) {return;}
+	var query = msg.content.split(" ");
+	if(!query){return;}
+	var command = query[0].toLowerCase();
+	var parameter;
+	if(query.length > 1) {
+		parameter = query.slice(1).join(" ");
+	}else{
+		parameter = "";
```

Comment:
I changed `parameter` to an `inline if` and made it a `const`. This should work too?